tests/testthat/_snaps/backend-mssql.md

```diff
@@ -96,22 +102,24 @@
 # handles ORDER BY in subqueries
 
     Code
-      sql_query_select(simulate_mssql(), "x", "y", order_by = "z", subquery = TRUE)
+      sql_query_select(simulate_mssql(), ident("x"), ident("y"), order_by = "z",
+      subquery = TRUE)
     Warning <warning>
       ORDER BY is ignored in subqueries without LIMIT
       i Do you need to move arrange() later in the pipeline or use window_order() instead?
     Output
-      <SQL> SELECT 'x'
-      FROM 'y'
+      <SQL> SELECT `x`
```

Comment:
Should the quoting have changed? Did you fix a bug?

Reply:
Ah, yes, there was a bug in the tests, passing bare characters instead of identifiers/sql, see

https://github.com/tidyverse/dbplyr/pull/642/files#diff-0a78a645632601539fd36938d193efcde6f89ef87f569165ee090508e4fe0474